web/src/invite.js

```diff
@@ -204,7 +210,13 @@ function valid_to(expires_in) {
     return $t({defaultMessage: "Expires on {date} at {time}"}, {date, time});
 }
 
-function get_expiration_time_in_minutes() {
+export function get_expiration_time_in_minutes(
```

Comment:
We should probably move this out to some other file, since having the realm deletion code import the invitation code is pretty clunky.